tests/CommonChunksTestOutputBuilder.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
this is an anonymous class in the original repo, which doesn't play well on PHP > 7